.appveyor.yml

```diff
@@ -0,0 +1,101 @@
+build: false
+shallow_clone: true
+platform:
+  - x64
+#matrix:
+#  fast_finish: true # kills the build at the first failure
+clone_folder: C:\projects\dbal
+
+## Build matrix for lowest and highest possible targets
+environment:
+  dependencies: current
+  matrix:
+  - db: mysql
+    php_ver_target: 7.0
+  - db: mysql
+    php_ver_target: 7.1
+  - db: mssql
+    php_ver_target: 7.0
+  - db: mssql
+    php_ver_target: 7.1
+  - db: mysqli
+    php_ver_target: 7.0
+  - db: mysqli
+    php_ver_target: 7.1
+  - db: sqlite
+    php_ver_target: 7.0
+  - db: sqlite
+    php_ver_target: 7.1
+  - db: pgsql
+    php_ver_target: 7.0
+  - db: pgsql
+    php_ver_target: 7.1
+
+## Cache composer bits
+#cache:
+#    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET PHP=1
+  - SET ANSICON=121x90 (121x90)
+services:
+  - mssql2014
+  - mysql
+  - postgresql94
+  - iis
+
+## Install PHP and composer, and run the appropriate composer command
+install:
+    - IF EXIST c:\tools\php (SET PHP=0)
```

Comment:
Do we really need this? PHP has to be installed on each build anyways, doesn't it? Can we get rid of the PHP variable?